_pytest/mark.py

```diff
@@ -63,7 +63,7 @@ def extract_from(cls, parameterset, legacy_force_tuple=False):
             argval = argval,
 
         if newmarks:
-            warnings.warn(MARK_PARAMETERSET_UNPACKING)
+            warnings.warn(MARK_PARAMETERSET_UNPACKING, stacklevel=2)
```

Comment:
in this case stacklevel is also not usable, its a case where its probably more sensible to warn explicit and pass the test around